dt/bmo01/dataplane/baremetalhosts/baremetalhost_template.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
```

Comment:
Would it be beneficial to add the annotation to disable inspection here (`inspect.metal3.io: disabled`)?  Or do you need that data for something later on?

Reply:
I am not using inspection data, so yes this would shave a couple of minutes of the run.
I will add it as `inspect.metal3.io: _replaced_` in the template, and set it based on values.